boards/arm/black_f407ve/black_f407ve.yaml

```diff
@@ -15,3 +15,4 @@ supported:
   - usb
   - spi
   - gpio
+vendor: st
```

Comment:
Vendor isn't st here, but I can't find a proper vendor.
I think we'd need a unknown, generic or whatever.. to identify such platforms

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/59650

Reply:
Dts fixed. Please update